tests/src/core/testqgsproject.cpp

```diff
@@ -597,6 +597,50 @@ void TestQgsProject::testSetGetCrs()
   ellipsoidChangedSpy.clear();
 }
 
+void 
```

Comment:
I don't think tests run like this -- I think they need to be defined as private slots in order for the test executable to run them

Reply:
ACtually this test definitely isn't running - because it's testing @map_crs, etc, when it should be testing project_crs,...!

Reply:
Sorry too many windows opened and ended up commit this instead of something else.